src/uu/cp/Cargo.toml

```diff
@@ -44,5 +44,7 @@ name = "cp"
 path = "src/main.rs"
 
 [features]
+default = ["cli-parser"]
 feat_selinux = ["selinux"]
 feat_acl = ["exacl"]
+cli-parser = ["clap", "uucore/cli-parser"]
```

Comment:
Can we make it `dep:clap`? I think that's the preferred syntax for enabling optional dependencies now.